content/300-guides/300-upgrade-guides/200-upgrading-versions/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: 'Upgrading to latest'
-metaTitle: 'Upgrading to latest (Guides)'
+title: 'Upgrading versions'
```

Comment:
This renaming needs a redirect, same for the existing subpages (codemods I think) @molebox

Reply:
I've made the redirect for this 😊